File: src/main/java/com/lord/small_box/dtos/DepositItemComparatorDto.java
package com.lord.small_box.dtos;

import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class DepositItemComparatorDto {

	
	private ExcelItemDto excelItemDto;
	
	private List<PurchaseOrderItemCandidateDto> purchaseOrderItemCandidateDtos;
}

File: front/small-box/src/app/models/depositItemComparatorDto.ts
import { ExcelItemDto } from "./excelItemDto";
import { PurchaseOrderItemCandidateDto } from "./purchaseOrderItemCandidateDto";

export class DepositItemComparatorDto{
    excelItemDto!:ExcelItemDto;
    purchaseOrderItemCandidateDtos:PurchaseOrderItemCandidateDto[]=[];
}